Formula/irrlicht.rb

```diff
@@ -43,10 +44,25 @@ def install
         "#  error ZLIB_VERNUM != PNG_ZLIB_VERNUM \\",
         "#  warning ZLIB_VERNUM != PNG_ZLIB_VERNUM \\"
 
+      extra_args = []
+
+      # Fix "Undefined symbols for architecture arm64: "_png_init_filter_functions_neon"
+      # Reported 18 Nov 2020 https://sourceforge.net/p/irrlicht/bugs/452/
+      extra_args << "GCC_PREPROCESSOR_DEFINITIONS='PNG_ARM_NEON_OPT=0'" if Hardware::CPU.arm?
+
+      xcodebuild "-project", "source/Irrlicht/MacOSX/MacOSX.xcodeproj",
+                 "-configuration", "Release",
+                 "-target", "IrrFramework",
```

Comment:
Do they not have a shared library target?

Reply:
Nope, but we could do [what the MacPorts project does](https://github.com/macports/macports-ports/blob/master/devel/irrlicht/Portfile#L71) and add a symlink:

```rb
frameworks.install "source/Irrlicht/MacOSX/build/Release/IrrFramework.framework"
ln_s frameworks/"IrrFramework.framework/Versions/A/IrrFramework", lib/"libIrrlicht.dylib"
```

Reply:
Yea, may as well:
```ruby
include.install_symlink frameworks/"IrrFramework.framework/Versions/A/Headers" => "irrlicht"
lib.install_symlink frameworks/"IrrFramework.framework/Versions/A/IrrFramework" => "libIrrlicht.dylib"
```

Reply:
Oh, wait, you've already installed the headers into `include`.

Reply:
Cancelled the merge so that we can make your changes here.